config.toml

```diff
@@ -44,3 +44,6 @@ consumersecret    = ""
 accesstoken       = ""
 accesstokensecret = ""
 
+[youtube]
+key               = ""
```

Comment:
I could add a comment or something on where to get it.

See the API Key section. It's specifically for youtube access.
https://developers.google.com/youtube/registering_an_application